tardis/visualization/tools/sdec_plot.py

```diff
@@ -1700,14 +1700,16 @@ def _plot_emission_ply(self):
             )
 
         # Contribution from each element
-        for species_counter, identifier in enumerate(self.species):
+        for (species_counter, identifier), species_name in zip(
+            enumerate(self.species), self._species_name
+        ):
             try:
                 self.fig.add_trace(
                     go.Scatter(
                         x=self.emission_luminosities_df.index,
                         y=self.emission_luminosities_df[identifier],
                         mode="none",
-                        name="none",
+                        name=species_name + " Emission",
                         fillcolor=self.to_rgb255_string(
                             self._color_list[species_counter]
                         ),
```

Comment:
I changed this because: 

1. It replaces `none` with the name of the element on which the cursor is being hovered, as mentioned in this issue:  #1670
2. It allows me to save traces by their names in the `hdf` file I use for testing. I was previously saving the traces using their index number, but I thought that saving them by their name is more maintainable. 

Is this change alright?